libraries/botbuilder-dialogs-adaptive/src/recognizers/multiLanguageRecognizer.ts

```diff
@@ -36,6 +43,11 @@ export class MultiLanguageRecognizer extends AdaptiveRecognizer implements Multi
         }
     }
 
+    /**
+     * @param dialogContext The dialog context.
+     * @param activity The activity
+     * @returns A promise representing the asynchronous operation.
+     */
```

Comment:
Dont invent new descriptions, use the ones from DotNet, also it doesnt have a description
![imagen](https://user-images.githubusercontent.com/62260472/150148545-ea4a632a-a422-47b9-ac62-32d5c6e68c35.png)